src/index.html

```diff
@@ -1,13 +1,416 @@
 <!DOCTYPE html>
-<html lang="en">
+<html lang="en" class="page">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
+    <title>BOSE</title>
+    <link
+      href="./images/bose/sound_waves.png"
+      class="page__favicon"
+      rel="fav"
+      type="image"
+    >
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&display=swap"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
     <script src="scripts/main.js" defer></script>
   </head>
-  <body>
-    <h1>Miami</h1>
+  <body class="page__body">
+    <header class="header page__section">
+      <div class="container">
+        <div class="header__content">
+          <div class="top-actions">
+
+            <a href="#" class="logo">
+              <img
+                class="header__logo-image"
+                src="./images/bose/logo.png"
+                alt="bose logo"
+              >
+            </a>
+
+            <a href="tel:+1 234 5555-55-55" class="header__call icon icon--phone"></a>
+            <a href="#menu" class="icon icon--menu"></a>
+
+          </div>
+
+          <h1 class="header__title">
+            The world<br>shades.
+            <br>
+            Your music<br>shines.
+          </h1>
```

Comment:
It's impossible to read what is written in header, try to work with your background
![image](https://user-images.githubusercontent.com/37161315/170449558-21d72e45-d65b-4f4a-b8cf-cf19c2f6e539.png)